src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.cs

```diff
@@ -416,7 +416,7 @@ private TryUpdateSourceResults TryUpdateSource()
                 return TryUpdateSourceResults.NotUpdated;
             }
 
-            if (selectedPackageSource.Name.Equals(name, StringComparison.CurrentCultureIgnoreCase) && selectedPackageSource.Source.Equals(source, StringComparison.OrdinalIgnoreCase))
```

Comment:
I decided not to make this change in this file, revert the changes. It brings some unwanted side effects, if update existing source `encyclopaedia` with `Encyclopædia` on PMUI then save it creates duplicate entrees in nuget.config file.
Problem is when saving it check existing using packagesource key but due existing following code it assumes above 2 are different hence creates another entry instead of updating it. https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/Settings/Items/SourceItem.cs#L80